dZoneManager/Zone.cpp

```diff
@@ -539,8 +539,13 @@ void Zone::LoadPath(std::ifstream& file) {
 					BinaryIO::BinaryRead(file, character);
 					value.push_back(character);
 				}
-				LDFBaseData* ldfConfig = LDFBaseData::DataFromString(parameter + "=" + value);
-				waypoint.config.push_back(ldfConfig);
+				LDFBaseData* ldfConfig = nullptr;
```

Comment:
Movement path waypoint configs to not have type data, so just make everything a string and let someone else deal with it when they need to use it